HIRS_AttestationCA/src/main/java/hirs/attestationca/service/SupplyChainValidationServiceImpl.java

```diff
@@ -288,6 +303,41 @@ private SupplyChainValidation validatePlatformCredentialAttributes(final Platfor
         }
     }
 
+    private SupplyChainValidation validateDeltaPlatformCredentialAttributes(
+            final PlatformCredential delta,
+            final DeviceInfoReport deviceInfoReport,
+            final PlatformCredential base) {
+    /*
+     * Do we need a new ValidationType for deltas?
+     */
+        final SupplyChainValidation.ValidationType validationType =
+                SupplyChainValidation.ValidationType.PLATFORM_CREDENTIAL_ATTRIBUTES;
+
+        if (delta == null) {
```

Comment:
Should you also check if the deviceInfoReport is null? Not sure how much "defensive programming" is expected/recommended

Reply:
The AbstractACA class does a null check on the deviceInfoReport before it is rolled into a Device object and passed to SupplyChainValidationService.